org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/ReferencesHandlerTest.java

```diff
@@ -175,4 +177,19 @@ public void testDeclarationInReferences() throws Exception {
 		assertEquals(new Range(new Position(14, 15), new Position(14, 25)), l.getRange());
 	}
 
+	// https://github.com/eclipse/eclipse.jdt.ls/issues/2405
+	@Test
+	public void testReferencesInJRE() throws Exception {
```

Comment:
Is this test valid, because even without this fix, System.out worked for me for some reason. May be we should try a class which is not in java.lang ?

Reply:
You can try to comment ReferencesHandler.java, lines 65-67 and run the test.

Reply:
I guess what happened in my workspace could be a fluke. Lets keep this.